eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -242,9 +231,25 @@ function Upload-Blobs
         & $($AzCopy) cp "$($DocDir)/**" "$($DocDest)/$($PkgName)/latest$($SASKey)" --recursive=true
     }
 }
+
+function RetrieveReleaseTag([Object[]]$pkgs) {
+    if (!$pkgs -or !$pkgs[0]) {
+        Write-Warning "There is no release tag retrieved out from current package. "
+        return ""
+    }
+    return $pkgs[0].Tag
+}
+
 # VERIFY PACKAGES
 $apiUrl = "https://api.github.com/repos/$repoId"
-$pkgList = VerifyPackages -pkgRepository $packageRepository -artifactLocation $artifactLocation -workingDirectory $workingDirectory -apiUrl $apiUrl -releaseSha $releaseSha -continueOnError $continueOnError
+$pkgList = VerifyPackages -pkgRepository $packageRepository 
+-artifactLocation $artifactLocation 
```

Comment:
this multi-line function isn't going to work if you want to do this you need to add a back tick to the end of each line.

Reply:
I copied this from update-doc-metadata.ps1. :) Will test this.

Reply:
Missed back tick when copy. :) Fixed